### PR TITLE
UI Polish

### PR DESCRIPTION
**Pull request by rsgc-peterson-e**
VELO-1504

### CODE REVIEW

**Comment by rsgc-peterson-e** [comment]
Screen Shot 2019-08-12 at 12 00 39 PM
Screen Shot 2019-08-12 at 11 55 33 AM

File: web/src/App.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ import axios from 'axios';
 import { CircularProgress } from '@material-ui/core';
 import urljoin from 'url-join'
 
-const API_ADDRESS = urljoin(window.location.protocol + '//' + window.location.host, 'api')
+// const API_ADDRESS = urljoin(window.location.protocol + '//' + window.location.host, 'api')
+const API_ADDRESS = 'http://localhost:8080/api'
```
Comment (crdueck) [inline web/src/App.js]: This shouldn't be committed right? Also, why do we need to change the API address now that we have skaffold for local testing?

File: web/src/components/GridCell.js [hunk 2]
```diff
@@ -32,7 +33,12 @@ class SingleGridCell extends React.Component {
   }
 
   cellClick = (event) => {
-    this.props.handleCellClick(event)
+    // this.props.handleCellClick(event)
```
Comment (crdueck) [inline web/src/components/GridCell.js]: Is this commented code still needed? If not, we should remove it

File: web/src/App.js (hunk 1 above)
Comment (rsgc-peterson-e) [reply web/src/App.js]: My bad, I will remove the test URL from the commit. I still use a local test server to see my changes in real time as the `npm build` and `skaffold` can take a long time.

File: web/src/components/GridCell.js (hunk 2 above)
Comment (rsgc-peterson-e) [reply web/src/components/GridCell.js]: Will remove.

**Pull request merged**